ai_ref_kits/visual_gen_studio/convert_and_optimize_texttoimage.py

```diff
@@ -0,0 +1,118 @@
+import argparse

```

Comment:
Let's name it text2image - more readable :P

Reply:
Updated the script name and also in readme instructions